File: app/src/main/java/com/BlizzardArmory/overwatch/Competitive/Games.java
package com.BlizzardArmory.overwatch.Competitive;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Games {

    @SerializedName("played")
    @Expose
    private int played;
    @SerializedName("won")
    @Expose
    private int won;

    public int getPlayed() {
        return played;
    }

    public void setPlayed(int played) {
        this.played = played;
    }

    public int getWon() {
        return won;
    }

    public void setWon(int won) {
        this.won = won;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/overwatch/Games.kt
package com.BlizzardArmory.model.overwatch

import com.google.gson.annotations.SerializedName


/**
 * The type Games.
 */
data class Games(

    @SerializedName("played")
    var played: Int,

    @SerializedName("won")
    var won: Int

)